test/Tizen.NUI.Samples/Tizen.NUI.Samples/Samples/WindowTest.cs

```diff
@@ -18,12 +18,20 @@ public class WindowTest : IExample
 
         int addingInput;
         Timer tm;
+        bool manualRotation;
+        int rotation_count;
```

Comment:
I think that rotationCount is better.

Reply:
I see, I will replace with rotationCount